builtin/logical/pki/path_root.go

```diff
@@ -243,14 +247,23 @@ func (b *backend) pathCASignIntermediate(
 	case "pem":
 		resp.Data["certificate"] = cb.Certificate
 		resp.Data["issuing_ca"] = cb.IssuingCA
+		resp.Data["issuing_ca_chain"] = cb.IssuingCAChain
 
 	case "pem_bundle":
-		resp.Data["certificate"] = fmt.Sprintf("%s\n%s", cb.Certificate, cb.IssuingCA)
+		resp.Data["certificate"] = cb.ToPEMBundle()
 		resp.Data["issuing_ca"] = cb.IssuingCA
+		resp.Data["issuing_ca_chain"] = cb.IssuingCAChain
 
 	case "der":
 		resp.Data["certificate"] = base64.StdEncoding.EncodeToString(parsedBundle.CertificateBytes)
 		resp.Data["issuing_ca"] = base64.StdEncoding.EncodeToString(parsedBundle.IssuingCABytes)
+		for _, cert := range parsedBundle.IssuingCAChainBytes {
```

Comment:
I believe this is one of those "reasons to use PEM" things. It may only be supported via PKCS12 or some such thing. If that's the case we should just note this as an explicit constraint and ignore the chain; or, use a slice type and put the chain in order in the slice, one b64 DER'd cert at a time.